pkg/httpcache_test.go

```diff
@@ -24,7 +24,7 @@ func TestHTTPCache_CacheDir(t *testing.T) {
 	cache := NewHTTPCache()
 
 	dir := cache.cacheDirFunc()
-	assert.Equal(t, "/foo/bar/.cache/helm-values-schema-json/httploader", dir)
+	require.Contains(t, dir, "/helm-values-schema-json/httploader")
```

Comment:
changed it like this because it fails on macOS. there the expected string is `/foo/bar/Library/Caches/helm-values-schema-json/httploader`.  another option for multi OS might be to consider using `assert.Regexp`